searchlib/src/vespa/searchlib/tensor/hnsw_index_base.h

```diff
@@ -72,29 +78,47 @@ class HnswIndexBase : public NearestNeighborIndex {
     using LinkArray = vespalib::Array<uint32_t>;
 
     const DocVectorAccess& _vectors;
+    RandomLevelGenerator& _level_generator;
     Config _cfg;
     NodeRefVector _node_refs;
     NodeStore _nodes;
     LinkStore _links;
     uint32_t _entry_docid;
+    uint32_t _entry_level;
```

Comment:
i found it easier to have level as "int" always.  that also allows _entry_level = -1 to signal no valid level.

Reply:
Fixed